applications/luci-app-packet-capture/htdocs/luci-static/resources/view/packet_capture/tcpdump.js

```diff
@@ -0,0 +1,293 @@
+'use strict';
+'require rpc';
+'require uci';
+'require ui';
+'require fs';
+'require form';
+'require network';
+'require tools.widgets as widgets';
+
+var eventSource,
+	captureFilePoll,
+	hostName;
+
+function stopTcpdump() {
+	fs.exec("/usr/sbin/packet_capture_stop").then(function(replay) {
+		if(eventSource) { 
+			eventSource.close();
+		}
+	}.bind(this)).catch(function(error) {
+		console.log(error);
+	});
+}
+
+window.onbeforeunload = stopTcpdump;
+
+var callLuciProcessList = rpc.declare({
+	object: 'luci',
+	method: 'getProcessList',
+	expect: { result: [] }
+});
+
+var callInitAction = rpc.declare({
+	object: 'luci',
+	method: 'setInitAction',
+	params: [ 'name', 'action' ],
+	expect: { result: false }
+});
+
+function findButton(title) {
+	var buttons = document.querySelectorAll('button');
```

Comment:
This will not work once `Download` is translated into other languages. I suggest to add a dedicated attribute to the button, e.g. `data-action="download"` which you can match. This would also simplify this function into `return document.querySelector('button[data-action="..."]');`